TG-Netflix/src/pages/discover/Discover.jsx

```diff
@@ -88,7 +88,7 @@ export default function Discover({ banner, categories, movie }) {
         )}
         {globalModalState.modalState && <CallBigModal movieID={movieID} />}
         <div className="fade-container">
-          {categoriesState.length > 0 && <LaneHandler categories={categories} movie={movie} />}
+          {categoriesState.length > 0 && <LaneHandler categories={categories} movie={movie} setIsHovering={setIsHovering}/>}
```

Comment:
Good fix! ik had niet gedacht dat we deze nog zouden oplossen, very nice. Ik bespeur geen ongewenste bijeffecten van de verandering. Approved!